### PR TITLE
Fixes for #184

### PR DESCRIPTION
**Pull request by madflow**
* 0 (Zeros) were treated as missing values - in a Spreadsheet these are perfectly valid values. empty() has been replaced with a helper method that honours zeros as valid values.

* The calculated values to show in a row were computed differently between ODS and XLSX/CSV. This was due to the attempt to hinder the ODS reader to create <=16384 empty cells when the file has been created by Excel (2010 ? ). This was done by convention - when the last cell (node) in a row was "empty" and the cell should be repeated (number-columns-repeated), than no values where pushed to the list. This convention had side effects on perfectly valid empty values, that should be repeated. I replaced the convention with a vendor specific ```hack``` - this feels unnecessary - but something's got to give when you are dealing with different vendors.

I worked with what I saw in the code - there can be side effects I do not know of.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @madflow has signed the CLA. Thanks for the pull request!

File: src/Spout/Reader/ODS/RowIterator.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ class RowIterator implements IteratorInterface
     const XML_NODE_TABLE = 'table:table';
     const XML_NODE_ROW = 'table:table-row';
     const XML_NODE_CELL = 'table:table-cell';
+    const MAX_COLUMNS_EXCEL= 16384;
```
Comment (adrilo) [inline src/Spout/Reader/ODS/RowIterator.php]: add a space between const and =

File: src/Spout/Reader/ODS/RowIterator.php [hunk 2]
```diff
@@ -127,22 +128,23 @@ public function next()
...
+                    $isEmptyRow = ($numCellsRead <= 1 && $this->isEmptyCellValue($cellValue));
                     if ($isEmptyRow) {
                         // skip empty rows
                         $this->next();
                         return;
                     }
 
-                    // Only add value if the last read cell is not empty or does not need to repeat cell values.
+                    // Only add the value if the last read cell is not a trailing empty cell repeater in Excel.
+                    // The current count of read columns is determined my counting the the values in $rowData.
```
Comment (adrilo) [inline src/Spout/Reader/ODS/RowIterator.php]: ... BY counting THE... (my => by, only one THE)

**Comment by adrilo** [comment]
Minor comments, otherwise looking good! Thanks @madflow

**Comment by adrilo** [comment]
Thanks!!

**Pull request merged**